scripts/local_test_cms_pipeline.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -e
+# set -e
```

Comment:
I would prefer to keep `set -e` to make sure the pipeline fails in case of any error.